Frontend/src/register/components/registration_list.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react'
+import deleteRegistration from '../../api/registration/delete/delete_registration'
+import getRegistrations from '../../api/registration/get/get_registrations'
+import updateRegistration from '../../api/registration/patch/update_registration'
+import styles from './list.module.scss'
+
+function StatusDropdown({ status, setStatus }) {
```

Comment:
Can we try to break up each component into it's own file?

Reply:
It's also standard to name the file after the default exported function, so `RegistrationList.jsx` in this case (as opposed to `registration_list.jsx`).